ansible/roles/test/tasks/sonic.yml

```diff
@@ -162,6 +162,10 @@
   include: pfc_wd.yml
   tags: pfc_wd
 
+- name: Test QoS Config and utility test
+  include: qos_config.yml
+  tags: qosconfig
+
```

Comment:
Can you also add an access point of this test case in test by test name infrastructure?

I think you need to change following file:

- https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/vars/testcases.yml
  This file organizes test cases in alphabetic order.You can following any existing test case as example.

Thanks, 
Ying